### DISCUSSION TITLE
[RELEASE] Elevator

### DISCUSSION THREAD

**Discussion by qloakongithub** [nzp-team/nzportable#1058] [Map Releases]
### Elevator

You've just accepted your first job as a repairman; at a hotel in Munich called Sterben Hotel. You were hired to board up the windows on the higher floors due to them not being used anymore. However, while on the elevator up, it suddenly stops on a maintenance floor. You look around, confused, and press the elevator buttons to go up. Instead, the door just opens, but jams, midway. You look through the door to see... zombies? Scared, you look around the elevator for a weapon, and find a pistol. You board up the windows and get ready to defend yourself against the increasing hoards of zombies.

This map is a survival map set within; you guessed it, an elevator. You must defend yourself from 2 sides, starting with nothing but a pistol and making your way up to the big guns. The map contains Juggernog, Quick Revive, a Mystery Box, and a Teddy Bear shaped Pack-a-Punch.

Version 1.0: Original version of map

[DOWNLOAD v1.0](https://archive.org/download/elevator-v-1/Elevator-v1.zip)

![Elevator](https://github.com/user-attachments/assets/fe853fb4-0977-4e5f-b882-e23f130d02b7)

![fte-20241223032759-0](https://github.com/user-attachments/assets/f76502bf-d129-4382-be1d-e4ad6467315b)

**Comment by qloakongithub** [comment]
Also this was done with a lack of sleep and probably has a few bugs. Sorry about that.

**Reply by mmkkiinn** [reply]
Why happining this 
![Screenshot_2024-12-23-13-12-24-26_2f85358b2198d26f8aca533d68bee793](https://github.com/user-attachments/assets/b9585e16-1851-43c2-aad4-245cefb6e25a)

**Reply by qloakongithub** [reply]
No idea but I'm gonna see if I can fix it
